### DISCUSSION TITLE
Disable Auto Scroll

### DISCUSSION THREAD

**Discussion by jasonlfunk** [spatie/ray#479] [Ideas]
Sometimes if my app is spitting out a ton of messages, I will want to scroll up to look at some of them. However, every time a new one appears it automatically scrolls to the bottom of the list. I would like for the app to not scroll automatically if I have manually scrolled up to look at something. The current behavior makes it very difficult to look at old messages unless there are no news ones coming in.

**Comment by jasonlfunk** [comment]
Oops, this is a duplicate of #362. Sorry! It can be closed. (But the feature should be implemented!)

**Reply by AdrianMrn** [reply]
It's already implemented in the settings 😁 

![image](https://user-images.githubusercontent.com/12762044/122891672-fc5b2280-d344-11eb-85eb-75ada32b6667.png)

**Reply by jasonlfunk** [reply]
@AdrianMrn But not yet released?

![image](https://user-images.githubusercontent.com/685188/122911133-a5af1200-d35f-11eb-8224-aff2c0324b71.png)

**Reply by AdrianMrn** [reply]
What version of Ray are you running, and on what OS?

**Reply by jasonlfunk** [reply]
1.14.7
Mac OS 11.4

**Reply by jasonlfunk** [reply]
I see how that this version is rather old. I guess the auto-updater broke in this version? It still says it's the latest when I try to update. I'll just download the new version.

**Reply by jasonlfunk** [reply]
@AdrianMrn Sorry, but it seems that 1.14.7 is the latest version available for Intel mac (downloaded from the website - https://spatie.be/docs/ray/v1/the-ray-desktop-app/download-the-free-demo) and from Homebrew.

**Reply by AdrianMrn** [reply]
Whoops, you're right. I tinkered with the release script a couple of days ago, I'll try to fix it and get a new version out.

**Reply by AdrianMrn** [reply]
There should be a 1.15.0 version available for download now (for all versions except m1). Could you please try it out?

**Reply by jasonlfunk** [reply]
It is still showing up as 1.14.7 for Mac Intel for me (from myray.app) and the updater says it's the latest version.

**Reply by AdrianMrn** [reply]
My mistake 🤦  The release is running again, please try again in about 10 minutes (around 16:10 CET).

**Reply by jasonlfunk** [reply]
Thanks! My 1.14.7 wouldn't auto-update. It said there was an update but didn't actually install/restart it. But that might just have been me. I downloaded it from the website and it works!

My only suggestion is that even if the auto-scroll is off, when I'm already scrolled to the bottom of it list, it should auto-scroll. But maybe that's too picky. :) (Or maybe at least an indicator that there is more entries below).

**Reply by AdrianMrn** [reply]
I agree with you and have tried to get this working, but ran into a lot of headache-material where the animation/smooth-scroll requires weird/ugly code to get it working, so I've put that code aside for a bit. We've also got some performance improvements to the list on our roadmap, which I'd like to get out before that change.

**Reply by genesiscz** [reply]
I'd love to see the button somewhere more available, because I feel a lot of people would use that frequently. Is that a possibility?

**Comment by jasonlfunk** [comment]
Sounds good. Just thought I’d mention it.

I really do love the app! :)

--
Jason Funk
